JSDemos/Demos/TreeList/Adaptability/description.md

```diff
@@ -1 +1,5 @@
-The TreeList can adapt its layout to&nbsp;narrow screens. If&nbsp;the screen width is&nbsp;not enough to&nbsp;display all columns, the component hides them according to&nbsp;the specified priority. To&nbsp;view the values of&nbsp;hidden columns, click the ellipsis button in&nbsp;the required row. Switch between the landscape and portrait layouts to&nbsp;see how the TreeList looks on&nbsp;wide or&nbsp;narrow screens.
\ No newline at end of file
+TreeList automatically adapts its layout to screens with different sizes. In this demo, you can switch between horizontal and vertical screen orientations. If columns do not fit the selected orientation, TreeList hides them one by one, starting with the rightmost column. Information from the hidden columns is still available in adaptive detail rows.
+
+To enable this feature, set the [columnHidingEnabled](/Documentation/ApiReference/UI_Components/dxTreeList/Configuration/#columnHidingEnabled) property to **true**.
+
+The TreeList hides columns based on their hiding priorities. Columns with lower priority values are hidden first. You can use the **columns[]**.[hidingPriority](/Documentation/ApiReference/UI_Components/dxTreeList/Configuration/columns/#hidingPriority) property to specify custom hiding priorities for those columns that you want to hide. Other columns will never be hidden.
```

Comment:
hiding priorities?

Reply:
Yes, they determine the order that TreeList uses to hide columns.

Reply:
ok, that's fine